CRM/Core/Page/EntityPageTrait.php

```diff
@@ -109,6 +109,7 @@ public function getContactId() {
   public function setContactId($contactId) {
     $this->_contactID = $contactId;
     $this->_contactId = $contactId;
+    $this->assign('contactId', $this->getContactId());
```

Comment:
It's not wrong but seems odd to call getContactId() here since already have the id - this was moved from below.

Reply:
I've become rather a fan of using getters as much as possible rather than passing around variables. This is rather in response to the fact that variables get passed around a LOT in civi and they can become quite mystical whereas getters always do the same thing wherever they live (although I have a suspicion this getter might be a bit odd)